backend/app/tasks/subscription_tasks.py

```
@@ -337,6 +337,95 @@ def _bind_task_to_subscription(db: Session, subscription: Any, task_id: int) ->
+    # Resolve KB refs to actual knowledge IDs
+    contexts = []
+    for kb_ref in kb_refs:
+        # Query knowledge base by displayName (spec.name) since that's what frontend stores
+        # The kb_ref.name is the displayName, not Kind.name
+        kbs = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == user_id,
+                Kind.kind == "KnowledgeBase",
+                Kind.namespace == kb_ref.namespace,
+                Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)